addons/road-generator/nodes/road_container.gd

```diff
@@ -851,7 +851,7 @@ func rebuild_segments(clear_existing := false):
 				next_pt = null
 
 		if not prior_pt and not next_pt:
-			push_warning("Road point %s not connected to anything yet" % pt.name)
+			push_warning("Road point %s not connected to anything yet" % str(pt.get_parent().name + "/" + pt.name))
```

Comment:
Since we're already using string substitution, may as well embrace it fully:

```suggestion
			push_warning("Road point %s/%s not connected to anything yet" % [pt.get_parent().name, pt.name])
```